src/ensembl/src/content/app/species/sample-data.ts

```diff
@@ -40,66 +40,66 @@ export const sampleData: RawSpeciesStats = {
-      average_cds_length: 1139.02,
+      coding_genes: 20424,
+      average_genomic_span: 67123.2,
+      average_sequence_length: 3538.93,
+      average_cds_length: 1160.33,
       shortest_gene_length: 8,
       longest_gene_length: 2473539,
-      total_transcripts: 153909,
-      coding_transcripts: 100607,
-      transcripts_per_gene: 7.54,
-      coding_transcripts_per_gene: 4.93,
-      total_exons: 1150521,
-      total_coding_exons: 764520,
-      average_exon_length: 243.09,
-      average_coding_exon_length: 149.87,
-      average_exons_per_transcript: 7.48,
-      average_coding_exons_per_coding_transcript: 7.6,
-      total_introns: 996612,
-      average_intron_length: 6367.19
+      total_transcripts: 162478,
+      coding_transcripts: 106190,
+      transcripts_per_gene: 7.96,
+      coding_transcripts_per_gene: 5.2,
+      total_exons: 1268315,
+      total_coding_exons: 825666,
+      average_exon_length: 246.94,
+      average_coding_exon_length: 149.22,
+      average_exons_per_transcript: 7.81,
+      average_coding_exons_per_coding_transcript: 7.78,
+      total_introns: 1105837,
+      average_intron_length: 6237.5
     },
     non_coding_stats: {
-      non_coding_genes: 23985,
-      small_non_coding_genes: 4870,
-      long_non_coding_genes: 16895,
+      non_coding_genes: 23967,
+      small_non_coding_genes: 4863,
+      long_non_coding_genes: 16884,
       misc_non_coding_genes: 2220,
-      average_genomic_span: 22904.09,
-      average_sequence_length: 1084.17,
+      average_genomic_span: 23077.19,
+      average_sequence_length: 940.34,
       shortest_gene_length: 41,
       longest_gene_length: 1375317,
-      total_transcripts: 54482,
-      transcripts_per_gene: 2.27,
-      total_exons: 183012,
-      average_exon_length: 347.07,
-      average_exons_per_transcript: 3.36,
-      total_introns: 128530,
-      average_intron_length: 15929.64
+      total_transcripts: 54748,
+      transcripts_per_gene: 2.28,
+      total_exons: 184477,
+      average_exon_length: 347.04,
+      average_exons_per_transcript: 3.37,
+      total_introns: 129729,
+      average_intron_length: 15993.77
     },
     pseudogene_stats: {
       pseudogenes: 15217,
-      average_genomic_span: 4124.33,
-      average_sequence_length: 802.62,
+      average_genomic_span: 4218.48,
+      average_sequence_length: 726.42,
       shortest_gene_length: 23,
       longest_gene_length: 909387,
-      total_transcripts: 18474,
-      transcripts_per_gene: 1.21,
-      total_exons: 43639,
-      average_exon_length: 356.59,
-      average_exons_per_transcript: 2.36,
-      total_introns: 25165,
-      average_intron_length: 4336.85
+      total_transcripts: 18617,
+      transcripts_per_gene: 1.22,
+      total_exons: 44532,
+      average_exon_length: 353.34,
+      average_exons_per_transcript: 2.39,
+      total_introns: 25915,
+      average_intron_length: 4372.89
     },
     homology_stats: {
-      coverage: 98.8
+      coverage: 86
```

Comment:
Surprised the homology stats a) reduced and b) by over 10%. Is this number correct?